new_dawn_server/users/models.py

```diff
@@ -7,6 +7,7 @@
 # An account model
 class Account(models.Model):
     birthday = models.DateField(blank=True, null=True)
+    city_preference = models.ForeignKey("locations.CityPreference", blank=True, null=True, on_delete=models.SET_NULL)
```

Comment:
So actually this shouldn't be a foreign key.
Account is a foreign key of city preference but not vice versa.
Just remove city preference. You don't need to explicitly specify it in this model if city preference already has foreign key set to account.

Same for answer_questions field in Profile. But don't worry about it now, I will put a PR to show how this works.

Reply:
You can make both of this and the city preference in Account to be ManyToMany field